creator-node/src/diskManager.ts

```diff
@@ -583,6 +571,116 @@ async function _migrateDirsWithLegacyStoragePaths(
   }
 }
 
+async function _migrateFileWithCustomStoragePath(
+  fileRecord: {
+    storagePath: string
+    multihash: string
+    cnodeUserUUID: string
+    dirMultihash?: string
+    fileName?: string
+    fileUUID: string
+    trackBlockchainId?: number
+  },
+  logger: Logger
+) {
+  const fetchStartTime = getStartTime()
+  const replicaSet = await getReplicaSetEndpointsByCnodeUserUuid({
+    libs: await initAudiusLibs({ logger }),
+    cnodeUserUuid: fileRecord.cnodeUserUUID,
+    parentLogger: logger
+  })
+  // Will retry internally
+  const { error, storagePath } = await fetchFileFromNetworkAndSaveToFS(
+    {}, // libs param is unused, so empty object is sufficient
+    logger,
+    fileRecord.multihash,
+    fileRecord.dirMultihash,
+    [
+      replicaSet.primary,
+      replicaSet.secondary1,
+      replicaSet.secondary2
+    ] /** targetGateways */,
```

Comment:
@SidSethi wondering if it might be worth it to remove this and just spray and pray (i.e., pass empty targetGateways array) since with the current code we have to:
* init libs, which I think queries every node for health
* query db to get wallet for given cnodeUserUUID
* query DN to get userId from wallet
* query DN to get replica set from userId

Reply:
agree this doesn't seem worth it, as long as we run this migration slowly, i think it is safe to spray and pray